packages/xdl/src/UserSettings.ts

```diff
@@ -50,7 +51,7 @@ function userSettingsJsonFile(): JsonFile<UserSettingsData> {
 function recentExpsJsonFile() {
   // TODO(ville): Add array support to JsonFile.
   // @ts-ignore JsonFile doesn't officially support arrays, only objects
```

Comment:
Is this comment and the one above still relevant?

Reply:
Yes, still valid. Type safe support for top level arrays requires some work because some object specific operations don’t make sense (e.g. set a string key, merge). But `read` and `write` happen to work and this code makes use of that.